src/toolkits/drawing_classifier/data_preparation.cpp

```diff
@@ -172,7 +173,7 @@ flex_list ramer_douglas_peucker(
     return compressed_stroke;
 }
 
-flex_list simplify_drawing(flex_list raw_drawing, int row_number) {
+flex_list simplify_drawing(flex_list &raw_drawing, int row_number) {
```

Comment:
Can't use a `const flex_list &` here since we make calls to `mutable_get` on elements of `raw_drawing`.

Reply:
Can you add a comment as to how exactly raw_drawing is changed?  If it's not changed, this should be const flex_list& and the rest should be modified as needed. 

Hmm.  don't actually see any reason to use mutable_get below.  mutable_get should only be used when the contents are explicitly being modified, but it seems the access is purely read-only.